ansible/roles/eos/templates/t1-64-lag-tor.j2

```diff
@@ -32,6 +32,24 @@ ip route 172.16.{{ tornum }}.{{ subnet }}/32 {{ props.nhipv4 }}
 ipv6 route 20AC:10{{ '%02X' % tornum }}:0:{{ '%02X' % subnet }}::/64 {{ props.nhipv6 }}
 {% endfor %}
 !
+{% if 'vips' in host %}
+{% for subnet in host['vips']['ipv4']['range'] %}
+ip route {{ subnet }} {{ props.nhipv4 }}
+{% endfor %}
+{% set index = 1 %}
+{% for subnet in host['vips']['ipv4']['range'] %}
+ip prefix-list test_vip_{{ index }} seq 10 permit {{ subnet }}
+{% set index = index + 1 %}
+{% endfor %}
+{% set index = 1 %}
+{% for subnet in host['vips']['ipv4']['range'] %}
```

Comment:
you can use only one subnet loop and add ip route, prefix-list and route-map in the loop.

Reply:
I was thinking of it when I first built the template, since this template create the startup configure file directly, not configuration input sequence , and I looked at the actual EOS config, prefixes are in one section and ip route are in one section, I am not 100% sure if I mixed them would be OK, this way is more safe.